docs/http_api.rst

```diff
@@ -40,6 +40,13 @@ Channels
        to. May be ``null`` if not desired. Not supported by every channel.
    :param str mo_url:
        URL to call on incoming messages (mobile originated) from this channel.
+       One or both of mo_url or amqp_queue must be specified.
+   :param str amqp_queue:
+       AMQP queue to repost messages onto for mobile originated messages. One
+       or both of mo_url or amqp_queue must be specified. Sends inbound
+       (mobile originated) messages to '{amqp_queue}.inbound', outbound (mobile
+       terminated) messages from '{amqp_queue}.outbound', and events to
+       '{amqp_queue}.events'
```

Comment:
Should we mention that events are sent to event URLs too if those are specified?

I think we also need to move the MT bit into its own sentence because it's a bit confusing for me at the moment.